src/KOKKOS/domain_kokkos.h

```diff
@@ -88,6 +97,126 @@ Few<double,3> DomainKokkos::unmap(Few<double,3> prd, Few<double,6> h,
   return y;
 }
 
+
+KOKKOS_INLINE_FUNCTION
+Few<double, 3> DomainKokkos::remap(Few<double,3> prd,
+                                   Few<double,6> h, Few<double, 6> h_inv,
+                                   int triclinic,
+                                   Few<double,3> boxlo, Few<double,3> boxhi,
+                                   Few<double,3> boxlo_lambda,
+                                   Few<double,3> boxhi_lambda,
+                                   Few<double,3> prd_lambda,
+                                   Few <double,3> x, imageint &image,
+                                   int periodic_bits)
+{
+  Few<double, 3> lo, hi, period, lambda, coord;
+
+  imageint idim, otherdims;
+  // Sorry for making this look like Java. :(
+  auto add_few_3 = [](Few<double, 3> a, Few<double, 3> b) {
```

Comment:
I actually only use this lambda once so we might as well remove it and paste the code verbatim. Will be more readable too.